coalib/output/ConsoleInteraction.py

```diff
@@ -227,7 +229,11 @@ def print_result(console_printer,
     console_printer.print(format_lines("[{sev}] {bear}:".format(
         sev=RESULT_SEVERITY.__str__(result.severity), bear=result.origin)),
         color=RESULT_SEVERITY_COLORS[result.severity])
-    console_printer.print(format_lines(result.message), delimiter="\n")
+    lexer = TextLexer()
+    result.message = highlight(result.message, lexer,
+                               TerminalTrueColorFormatter(
+                                    style=BackgroundMessageStyle))[:-1]
```

Comment:
add a third lexer parameter (optional) to your lambda so you can use it here as well?